docs/changelog/2238.misc.rst

```diff
@@ -0,0 +1 @@
+Omit backports.entry_points_selectable for Python >=3.10
```

Comment:
Can you add this as a feature please and not misc; misc basically reserved for project maintenance files only, not that affects code.

Reply:
Also, can we drop `backports.entry_points_selectable` entirely here?

Reply:
I suppose we can if we require `importlib-metadata>=3.6;python_version<"3.10"` and do:

```python
if sys.version_info >= (3, 10):
    from importlib.metadata import entry_points
else:
    from importlib_metadata import entry_points
```

-- not sure if that is much better.

Reply:
Alternatively can we use the old API on old pythons and the new API on new Pythons? Rather than try to use the same?

Reply:
Like this? https://github.com/pypa/virtualenv/pull/2241

Reply:
Added as feature. The other PR would fit my needs as well.

Reply:
I prefer 2241, so will be closing this.